Formula/minikube.rb

```diff
@@ -0,0 +1,28 @@
+class Minikube < Formula
+  desc "Tool that makes it easy to run Kubernetes locally"
+  homepage "https://kubernetes.io/docs/getting-started-guides/minikube/"
+  url "https://github.com/kubernetes/minikube.git",
+    :tag => "v0.26.1",
+    :revision => "6ded2b647d1b1f62100c630bcfcc1363c631ce2d"
+
+  # Docker required for non-Linux builds
+  depends_on "docker" => :build unless MacOS.version == Version::NULL
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    repo_path = buildpath/"src/k8s.io/minikube"
+    repo_path.install Dir["*"]
+
+    cd repo_path do
+      system "make"
+      bin.install "out/minikube" => "minikube"
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/minikube version 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
If you don't mind, would you point me to an example you'd like me to follow? Lots of the formulas I looked at do more or less exactly this.

Reply:
Look at `kube-aws` or `kubeless`